Alena/HTML_CSS/index.html

```diff
@@ -0,0 +1,535 @@
+                                <img src="images/arrow.png" alt="">
+                            </div>
+                        </div>
+                    </a>
+                </div>
+                <div class="prod2_block2">
+                    <div class="prod2_1">
+                        <div class="category">
+                            <p>Vegetable</p>
+                        </div>
+                        <img src="images/prod_4.png" alt="">
+                        <div class="prod_text">
+                            <p>Vegan Red Tomato</p>
+                            <div class="line"></div>
+                            <div class="prod_text_down">
+                                <div class="price">
+                                    <p class="price_1">$ 14.00 USD</p>
+                                    <p class="price_2">$ 9.37 USD</p>
+                                </div>
+                                <img src="images/stars.png" alt="">
+                            </div>
+                        </div>
+
+                    </div>
+                    <div class="prod2_2">
+                        <div class="category">
+                            <p>Fruits</p>
+                        </div>
+                        <img src="images/prod_5.png" alt="">
+                        <div class="prod_text">
+                            <p>Organic Almonds</p>
+                            <div class="line"></div>
+                            <div class="prod_text_down">
+                                <div class="price">
+                                    <p class="price_1">$ 21.00 USD</p>
+                                    <p class="price_2">$ 18.00 USD</p>
+                                </div>
+                                <img src="images/stars.png" alt="">
+                            </div>
+
+                        </div>
+                    </div>
+                    <div class="prod2_3">
+                        <div class="category">
+                            <p>Vegetable</p>
+                        </div>
+                        <img src="images/prod_2.png" alt="">
+                        <div class="prod_text">
+                            <p>Fresh Corn</p>
+                            <div class="line"></div>
+                            <div class="prod_text_down">
+                                <div class="price">
+                                    <p class="price_1">$ 21.23 USD</p>
+                                    <p class="price_2">$ 18.00 USD</p>
+                                </div>
+                                <img src="images/stars.png" alt="">
+                            </div>
+
+                        </div>
+
+                    </div>
+
+
+                </div>
+
+            </section>
+            <section class="eco">
+                <img src="images/eco_bg.png">
+                <div class="eco_block">
+                    <div class="eco_titles">
```

Comment:
add container for items. delete margin-top: 39.89px;
    margin-left: 60px; in eco_titles